src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/DepartureTransformer.kt

```diff
@@ -0,0 +1,51 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer
+
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.health.api.model.Departure
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.DepartureEntity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.DepartureReason
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.MoveOnCategory
+
+@Component
+class DepartureTransformer(private val destinationProviderTransformer: DestinationProviderTransformer) {
+  fun transformJpaToApi(jpa: DepartureEntity?): Departure? {
+    if (jpa == null) return null
+
+    return Departure(
+      bookingId = jpa.booking.id,
+      dateTime = jpa.dateTime,
+      reason = transformDepartureReason(jpa.reason),
```

Comment:
Would we return these as a JSON object, now we're using database entried?

Reply:
Yeah that's right, I've just pushed the change to return objects